hardware_layer/hardware_imu/src/dcm.cpp

```diff
@@ -31,7 +31,7 @@ int main(int argc, char **argv)
   ros::NodeHandle n;
   ros::Publisher out_pub = n.advertise<sensor_msgs::Imu>("imuyrp", 1000);
   ros::NodeHandle nh;
-  ros::Publisher chatter_pub = nh.advertise<std_msgs::Float64>("yaw", 1000);
+  ros::Publisher chatter_pub = nh.advertise<std_msgs::Float64>("/varun/sensors/imu/imu_raw", 1000);
```

Comment:
plz make this `/varun/sensors/imu/yaw` as I can't control `/imu_raw` topic in the case of gazebo. I will instead make a node which takes this message from gazebo and publishes on `varun/sensors/imu/yaw`.